.github/pull_request_template.md

```
@@ -0,0 +1,40 @@
+```sh
+cmake --build ./build -j$(nproc)
+
+# Unit tests
+./build/bin/unittests
+
+# Integration tests
+./build/bin/stress-tests ./stress-tests
+```
+
+<!-- If you added new tests, name them here: -->
+New tests:
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)